gapil/template/constant_sets.go

```diff
@@ -137,6 +137,9 @@ func (nl *nodeLabeler) addLabels(n semantic.Node, v analysis.Value) {
 	if !ok || len(ev.Labels) == 0 {
 		return // Parameter wasn't an enum, or had no labels.
 	}
+	if n == nil {
```

Comment:
I think the proper solution here is in 
```
 func (nl *nodeLabeler) traverse(n semantic.Node, v analysis.Value) {
```
traverse with `(n, k)  and (n, v)` instead of `(nil, k), (nil, v)`.